[([31mE[m)]

ERROR in :ddd.exception-outside-is/exception-outside-is-test (exception_outside_is.clj:4)
Uncaught exception, not in assertion.
Exception: java.lang.Exception: booo
 at ddd.exception_outside_is$fn__15611.invokeStatic (exception_outside_is.clj:4)
    ddd.exception_outside_is/fn (exception_outside_is.clj:4)
    kaocha.plugin.capture_output$eval2677$fn__2678$fn__2691$fn__2693$fn__2694.invoke (capture_output.clj:88)
    kaocha.type.var$eval15284$fn__15286$fn__15292.invoke (var.clj:28)
    kaocha.type.var$eval15284$fn__15286.invoke (var.clj:25)
    ...
    kaocha.plugin.capture_output$eval2677$fn__2678$fn__2683$fn__2684$fn__2685.invoke (capture_output.clj:79)
    ...
    kaocha.plugin.capture_output$eval2677$fn__2678$fn__2683$fn__2684.doInvoke (capture_output.clj:79)
    ...
    kaocha.testable$run.invokeStatic (testable.clj:92)
    kaocha.testable$run.invoke (testable.clj:83)
    kaocha.testable$run_testables.invokeStatic (testable.clj:123)
    kaocha.testable$run_testables.invoke (testable.clj:113)
    ...
    kaocha.type.ns$run_tests$fn__15350.invoke (ns.clj:53)
    ...
    kaocha.type.ns$run_tests.invokeStatic (ns.clj:53)
    kaocha.type.ns$run_tests.invoke (ns.clj:49)
    kaocha.type.ns$eval15354$fn__15355.invoke (ns.clj:71)
    ...
    kaocha.plugin.capture_output$eval2677$fn__2678$fn__2683$fn__2684$fn__2685.invoke (capture_output.clj:79)
    ...
    kaocha.plugin.capture_output$eval2677$fn__2678$fn__2683$fn__2684.doInvoke (capture_output.clj:79)
    ...
    kaocha.testable$run.invokeStatic (testable.clj:92)
    kaocha.testable$run.invoke (testable.clj:83)
    kaocha.testable$run_testables.invokeStatic (testable.clj:123)
    kaocha.testable$run_testables.invoke (testable.clj:113)
    kaocha.type.clojure.test$eval15556$fn__15557.invoke (test.clj:16)
    ...
    kaocha.plugin.capture_output$eval2677$fn__2678$fn__2683$fn__2684$fn__2685.invoke (capture_output.clj:79)
    ...
    kaocha.plugin.capture_output$eval2677$fn__2678$fn__2683$fn__2684.doInvoke (capture_output.clj:79)
    ...
    kaocha.testable$run.invokeStatic (testable.clj:92)
    kaocha.testable$run.invoke (testable.clj:83)
    kaocha.testable$run_testables.invokeStatic (testable.clj:123)
    kaocha.testable$run_testables.invoke (testable.clj:113)
    kaocha.api$run$fn__5498.invoke (api.clj:82)
    ...
    kaocha.api$run.invokeStatic (api.clj:74)
    kaocha.api$run.invoke (api.clj:60)
    kaocha.runner$_main_STAR_.invokeStatic (runner.clj:130)
    kaocha.runner$_main_STAR_.doInvoke (runner.clj:72)
    ...
    kaocha.runner$_main.invokeStatic (runner.clj:139)
    kaocha.runner$_main.doInvoke (runner.clj:137)
    ...
    clojure.main$main_opt.invokeStatic (main.clj:378)
    clojure.main$main_opt.invoke (main.clj:374)
    clojure.main$main.invokeStatic (main.clj:485)
    clojure.main$main.doInvoke (main.clj:448)
    ...
    clojure.main.main (main.java:37)
[31m1 test vars, 1 assertions, 1 errors, 0 failures.[m
